src/pages/match/[id]/view.tsx

```diff
@@ -1,68 +1,182 @@
 //third-party import
-import { getMatchById } from "@/lib/actions/match";
-import Database from "@/lib/resources/database";
-import { Match } from "@/lib/types/Match";
```

Comment:
things should be in local import:

- Database
- Match

Reply:
This is deleted, already moved and corrected the imports